Taskfile.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+
+dotenv: ['.env']
+
+tasks:
```

Comment:
It is generally a good practice to specify the context within your .env file explicitly when defining environment-dependent variables. This makes it easier for anyone new to the project to understand what variables are necessary and where they might come from.